DataFormats/Detectors/EMCAL/src/DataFormatsEMCALLinkDef.h

```diff
@@ -19,12 +19,16 @@
 #pragma link C++ class o2::emcal::Cell + ;
 #pragma link C++ class o2::emcal::Digit + ;
 #pragma link C++ class o2::emcal::Cluster + ;
+#pragma link C++ class o2::emcal::MCLabel + ;
 
 #pragma link C++ class std::vector < o2::emcal::TriggerRecord > +;
 #pragma link C++ class std::vector < o2::emcal::Cell > +;
 #pragma link C++ class std::vector < o2::emcal::Digit > +;
 #pragma link C++ class std::vector < o2::emcal::Cluster > +;
 
+#include "SimulationDataFormat/MCTruthContainer.h"
```

Comment:
I think it is better to have this include in your MCLabel class

Reply:
I think it's ok to have it here. We have done this elsewhere as well and MCLabel probably does not depend on the Container.